kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -54,7 +54,14 @@ data:
           # Requires ContainerLogV2 schema to be enabled. See https://aka.ms/ContainerLogv2 for more details.
           # enabled = "false"
           # stacktrace_languages = []
-
+       #[log_collection_settings.metadata_collection]
+          # if enabled will collect kubernetes metadata for ContainerLogv2 schema. Default is false.
```

Comment:
kube_meta_cache_ttl entry here  with comments about the default value and link to fluent-bit doc about this field.

Reply:
added